WcaOnRails/app/controllers/regulations_controller.rb

```diff
@@ -3,9 +3,26 @@
 class RegulationsController < ApplicationController
   include HighVoltage::StaticPage
 
+  before_action :ensure_trailing_slash
+
   private def page_finder_factory
     IndexPageFinder
   end
+
+  private def ensure_trailing_slash
+    desired_url = url_for(params.permit!.merge(trailing_slash: true))
```

Comment:
The fix was to just add a call to `permit!` here, which the internet warns is a security vulnerability, but I think we're actually fine here, as a redirect is going to send us back through our whole application flow where we'll have to explicitly permit parameters before using them.